src/aui/auibook.cpp

```diff
@@ -3359,10 +3359,35 @@ void wxAuiNotebook::SetPageSize (const wxSize& WXUNUSED(size))
     wxFAIL_MSG("Not implemented for wxAuiNotebook");
 }
 
-int wxAuiNotebook::HitTest (const wxPoint& WXUNUSED(pt), long* WXUNUSED(flags)) const
+int wxAuiNotebook::HitTest (const wxPoint &pt, long *flags) const
 {
-    wxFAIL_MSG("Not implemented for wxAuiNotebook");
-    return wxNOT_FOUND;
+    wxWindow *w = NULL;
+    long position = wxBK_HITTEST_NOWHERE;
+    wxAuiPaneInfoArray& all_panes = const_cast<wxAuiManager&>(m_mgr).GetAllPanes();
+    size_t i, pane_count = all_panes.GetCount();
```

Comment:
```suggestion
    const size_t pane_count = all_panes.GetCount();
```

Reply:
Changed this. I would have even moved `GetCount` in the loop condition, as I thought I once read that the compiler even optimizes this away, when it sees that the result won't change during the loop. But I actually copied the code from another function in the same file and thought that should be the way to go :-)